src/frontend/magnify/src/components/rooms/RoomRow/RoomRow.tsx

```diff
@@ -95,6 +107,18 @@ export default function RoomRow({ room, baseJitsiUrl }: RoomRowProps) {
     });
   };
 
+  const copyLinkToClipboard = (): void => {
+    navigator.clipboard.writeText(process.env.REACT_APP_FRONTEND_DOMAIN + '/' + room.slug).then(
```

Comment:
Good idea but more simple : use the `window.location.origin` instead process.env.